swarm/api/config.go

```diff
@@ -57,31 +57,33 @@ type Config struct {
 	NetworkId   uint64
 	SwapEnabled bool
 	SyncEnabled bool
+	PssEnabled  bool
 	SwapApi     string
 	Cors        string
 	BzzAccount  string
 	BootNodes   string
 }
 
 //create a default config with all parameters to set to defaults
-func NewDefaultConfig() (self *Config) {
+func NewConfig() (self *Config) {
```

Comment:
Why rename this?

Reply:
Because all other corresponding swarm methods had already been renamed `NewDefault*` to `New*`. Why, I don't know. But I wanted to make this the same.

Reply:
I still do not understand why this has been renamed. The `config` has this overriding architecture where the default config is overridden by the env vars which are overridden by the command line params. `NewDefaultConfig()` IMHO makes this a bit explicit. It's a minor thing and I do not mind if it stays renamed, but I do think that keeping `Default` in it has its merits.

Reply:
We can rename them all to `NewDefault*` then?

Reply:
I am not sure which you are referring to to be able to give an informed opinion. All I have added have `NewDefault...()`

Reply:
ok, but previously existing `NewDefault*Params` have become `New*Params` in the rewrite. We should stick to one scheme.